### DISCUSSION TITLE
OS, Tools and Co

### DISCUSSION THREAD

**Discussion by jenslink** [becarpenter/book6#66] [General]
Where would you put operating specific things, e.g., how to configure IPv6 on Linux, macOS, Windows, … 

And is there room for a troubleshooting / tools chapter?

**Comment by becarpenter** [comment]
I'll take that backwards...

Maybe a troubleshooting/tools chapter would be better than the largely empty management/ops chapter? In any case, I think it's a good idea (and chapter numbering doesn't matter too much, as the book is unlikely to be read in sequence). I have some busy days coming, but I can create a blank chapter soon.

As for o/s specific information, I don't think we should try to compete with regular o/s documentation, but short sections with pointers would be excellent - and to my mind would fit well in a troubleshooting/tools chapter too.

**Reply by buraglio** [reply]
I agree, I don't think the OS specifics are worth the squeeze due to frequent changes and existing documentation, but I could probably be convinced.

**Reply by buraglio** [reply]
I also agree that a troubleshooting chapter would be immensely useful.

**Reply by buraglio** [reply]
Brian, I created the troubleshooting chapter templates.

**Reply by becarpenter** [reply]
Thanks. As soon as I mend the problem in makebook, I will run it to fix up the various pointers...

**Comment by becarpenter** [comment]
When checking on the new chapter, I realised that I'd carefully hidden the instructions about how to create a new chapter. They're at https://github.com/becarpenter/book6/blob/main/utilities/chapterReorg.md. I had to fix a couple of things but the new chapter is open for input.

I added a pointer to those hidden instructions in [the main instructions](https://github.com/becarpenter/book6/blob/main/99.%20Chapter%20Template/99.%20Chapter%20Template.md). Probably I need to document makeBook itself, I'll open an issue for that.
